src/main/java/com/faforever/client/chat/ChannelTabController.java

```diff
@@ -661,4 +662,16 @@ public void onPlayerOnline(PlayerOnlineEvent event) {
     ChatChannelUser chatUser = chatService.getChatUser(event.getPlayer().getUsername(), channel.getName());
     associateChatUserWithPlayer(event.getPlayer(), chatUser);
   }
+
+  public void toggleSidePane(ActionEvent actionEvent) {
+    if (channelTabScrollPaneVBox.isManaged()) {
+      toggleSidePaneButton.setText("");
+      channelTabScrollPaneVBox.setManaged(false);
+      splitPane.setDividerPositions(1.0);
+    } else {
+      toggleSidePaneButton.setText("");
+      channelTabScrollPaneVBox.setManaged(true);
+      splitPane.setDividerPositions(0.8);
+    }
```

Comment:
Maybe look into Bindings could be an example to learn them here

Reply:
I think the bind method only works when I want to copy a property value Do you mean using a changeListener?

Reply:
Binds.createStringBinding allows for custom Bindings